src/opcode.rs

```diff
@@ -48,46 +48,56 @@ mod sealed {
     }
 }
 
+/// Common Linux types not provided by libc.
 pub mod types {
     use crate::sys;
     use bitflags::bitflags;
     use std::os::unix::io::RawFd;
 
     pub use sys::__kernel_rwf_t as RwFlags;
+
     pub use sys::__kernel_timespec as Timespec;
 
-    /// Opaque types, you should use `libc::statx` instead.
+    /// Opaque types, you should use [`statx`](struct@libc::statx) instead.
     #[repr(C)]
     pub struct statx {
         _priv: (),
     }
 
-    /// Opaque types, you should use `libc::epoll_event` instead.
+    /// Opaque types, you should use [`epoll_event`](libc::epoll_event) instead.
     #[repr(C)]
     pub struct epoll_event {
         _priv: (),
     }
 
+    /// A file descriptor that has not been registered with io_uring. This can be slower than using
```

Comment:
> This can be slower than using [`Fixed`].

I suspect this is not true, maybe we can delete this.

Reply:
The manpage for `io_uring_register` says that registering file descriptors "greatly reduces per-I/O overhead", so I took that to mean that using registered file descriptors would be faster. Maybe we could make it more specifically about overhead though.

Reply:
We can mention in `Fixed` that it will reduce some overhead in some cases.